### ISSUE TITLE
Motion animation best practices?

### ISSUE THREAD

**Issue by JayPanoz** [daisy/kb#12]
Hi there. 

First and foremost, thanks for this great resource! There’s no doubt it’s been useful for a lot of authors, and at least I can say it helped me tremendously when I was doing e-production. 

This issue is more of a question/comment than a feature or bug request, but I believe it could be a good idea exposing such information in the knowledge base.

I may be wrong but there appears to be quite a huge gap between WCAG 2.0 and WCAG 2.1 when it comes to motion animation.

“Success Criterion 2.3.3 Animation from Interactions” has only [been added @ 2.1](https://www.w3.org/TR/WCAG21/#animation-from-interactions) it seems, but it kinda creates a huge loophole in the sense [reduced motion](https://webkit.org/blog/7551/responsive-design-for-motion/) is all the rage **right now**, with all major browsers implementing this media feature. 

As a disclaimer, I myself experienced vestibular disorders and can testify [animations can hurt or even ruin entire days](https://source.opennews.org/articles/motion-sick/), so I wouldn’t be against promoting this “issue” so that authors designing interactive EPUBs with motion be aware this is something to take into account. 

One example of a best practice would probably be [this Mozilla demo](http://devtoolschallenger.com), asking the user if they want to disable motion animations. 

I haven’t found anything about that except for metadata, but in practice, putting a little bit of effort could make at least some books more accessible – instead of just tagging the publication with motion hazard, because they could still work super fine in static form with a switch at the beggining of the book for instance. 

Motion animation is something Reading Systems are taking into account more frequently, given all the work being done there, so it’s kinda important there is some evangelisation in order for authors to know this actually is an issue. 

A Reading System is only as good as the contents it must handle and render, so it will be useless to a person with vestibular disorders if the interactive fixed-layout has motion animation that will make the user sick.

**Comment by mattgarrish** [comment]
Yes, dynamic content has been a bit problematic to tackle in the KB as it can take so many different forms at the implementation level. There are more general guidelines, like this, that we could do more to put forward to people, though.
